bindata/v3.11.0/kube-controller-manager/pod.yaml

```diff
@@ -8,6 +8,18 @@ metadata:
     kube-controller-manager: "true"
     revision: "REVISION"
 spec:
+  initContainers:
+  - name: wait-for-host-port
+    image: ${IMAGE}
+    imagePullPolicy: IfNotPresent
+    command: ['/usr/bin/timeout', '100', "/bin/bash", "-c"] # 60 sec for graceful termination, 35 for minimum-termination-duration
```

Comment:
@sttts I've copied 100 here, if you know any reason this should be shorter lemme know

Reply:
It can. We don't have graceful termination for kube-controller-manager. 10 or at most 30 secs would be a good number.